htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -373,6 +373,15 @@ UPDATE llx_propal SET date_signature = date_cloture WHERE date_signature IS NULL
 
 
 ALTER TABLE llx_product ADD COLUMN batch_mask VARCHAR(32) NULL;
+ALTER TABLE llx_product ADD COLUMN life_duration VARCHAR(6);
+ALTER TABLE llx_product ADD COLUMN control_periodicity VARCHAR(6);
+
+ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime;
```

Comment:
What is goal of this date ? What is translation in french ?

Reply:
scrapping_date = mise au rebut

Reply:
Ok for scrapping_date, we can keep it as it is.